### PR TITLE
Afl wrappers

### PR DESCRIPTION
**Pull request by kobrineli**

### CODE REVIEW

File: projects/openxlsx/Dockerfile [hunk 1]
```diff
@@ -29,11 +29,14 @@ RUN git checkout 3eb9c748e3ecd865203fb9946ea86d3c02b3f7d9
...
      fuzzer.cc \
+     afl.cc \
      sydr.cc \
      xlst.dict \
      /
 
 # Build fuzz targets.
 RUN ../build.sh
 
+RUN mkdir /corpus
```
[the remark this answers is not in the document]
Comment (kobrineli) [reply projects/openxlsx/Dockerfile]: done

File: projects/expat/Dockerfile [hunk 2]
```diff
@@ -31,9 +31,11 @@ WORKDIR expat
...
 # Copy build script and fuzz targets for libFuzzer and Sydr.
-COPY build.sh main.c *.dict ./
+COPY build.sh main.c afl.cc *.dict ./
 
 # Build fuzz targets.
 RUN ./build.sh
 
 WORKDIR /
+
+RUN mkdir corpus
```
Comment (SweetVishnya) [inline projects/expat/Dockerfile:41]: https://github.com/strongcourage/fuzzing-corpus/tree/master/xml

We may get corpus from here

File: projects/expat/xml_parse_fuzzer_UTF-8-afl++.toml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Copyright 2021 ISP RAS
```
Comment (SweetVishnya) [inline projects/expat/xml_parse_fuzzer_UTF-8-afl++.toml:1]: set 2022 in new files

Comment (kobrineli) [reply projects/expat/xml_parse_fuzzer_UTF-8-afl++.toml:1]: done

Comment (kobrineli) [reply projects/expat/xml_parse_fuzzer_UTF-8-afl++.toml]: done

File: projects/fmt/chrono_duration.toml [hunk 4]
```diff
@@ -16,7 +16,7 @@
 exit-on-time = 7200
 
 [sydr]
-args = "-l debug -s 90 -m 8192"
+args = "-l debug -m 8192"
```
Comment (anfedotoff) [inline projects/fmt/chrono_duration.toml:19]: -m 8192 is default could be removed and other cases

File: projects/openssl/targets/asn1_afl.c [hunk 5]
```diff
@@ -0,0 +1,396 @@
+/*
+ * Copyright 2016-2021 The OpenSSL Project Authors. All Rights Reserved.
+ * Modifications copyright (C) 2021 ISP RAS
```
Comment (anfedotoff) [inline projects/openssl/targets/asn1_afl.c:3]: 2022 and other cases

**Review by anfedotoff — APPROVED** [review]

**Pull request merged**